localization/ar_tag_based_localizer/README.md

```diff
@@ -0,0 +1,107 @@
+# AR Tag Based Localizer
+
+**ArTagBasedLocalizer** is a vision-based localization package.
+
+<img src="./doc_image/ar_tag_image.png" width="320px">
+
+This package uses [the Aruco library](https://index.ros.org/p/aruco/) to detect AR-Tags from camera images and calculates and publishes the pose of the ego vehicle based on these detections.
+The positions and orientations of the ArTags are assumed to be written in the LaneLet2 format.
+
+This package includes two nodes.
+
+- `ar_tag_based_localizer` : A node that detects AR-Tags from camera images and publishes the pose of the ego vehicle.
```

Comment:
Could you elaborate a bit more on the design of this algorithm? [YabLoc's explanation](https://github.com/autowarefoundation/autoware.universe/blob/main/localization/yabloc/README.md#principle) would be one of the ideal styles, but doesn't have to be that specific. Just one or two or more sentence would be enough IMO.

Just an idea, but maybe simply writing down the mathematics behind it (a simple linear algebra) could be helpful for beginners to understand.

Reply:
Added an image showing the simple principle.
https://github.com/autowarefoundation/autoware.universe/pull/4347/commits/e9fb437a603f23d9e6d4b5ae5a42b02b0efdeff9

Reply:
Perfect!! Thank you :pray: